pincer/objects/guild/permissions.py

```diff
@@ -101,3 +101,21 @@ def to_int(self) -> Tuple[int]:
                 deny |= enum.value
 
         return allow, deny
+    
+    @property
+    def allow(self) -> int:
+        allow = 0
+        for enum in PermissionEnums:
+            if getattr(self, enum.name.lower()):
+                allow |= enum.value
+                
+        return allow
+    
+    @property
+    def deny(self) -> int:
+        deny = 0
+        for enum in PermissionEnums:
+            if getattr(self, enum.name.lower()) is False:
```

Comment:
```suggestion
            if not getattr(self, enum.name.lower()):
```

Reply:
Since a permission can be None, this does not work, since `not None` is `True`